synapse/models/biz.py

```diff
@@ -0,0 +1,204 @@
+import synapse.lib.module as s_module
+
+'''
+Model elements related to sales / bizdev / procurement
+'''
+
+class BizModule(s_module.CoreModule):
+    def getModelDefs(self):
+        modl = {
+            'types': (
```

Comment:
Do we want to add a biz:proposal form (or something of the like) to capture rfp proposals?

Reply:
Eventually yea, but i left it TBD on purpose for now ( let it come back up later )